### PR TITLE
Release v1.20.0

### PR DESCRIPTION
**Pull request by FrancoisDevemy5**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@reachfive/identity-core",
-  "version": "1.19.0",
+  "version": "1.19.1",
```
Comment (StanWarz) [inline package.json:3]: I would have bumped the minor version since there's not only fixes in the PR.

**Review by StanWarz — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (FrancoisDevemy5) [reply package.json:3]: ok

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,12 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+## [1.19.1] - 2021-01-14
```
Comment (roxanemace) [inline CHANGELOG.md:10]: You forgot to update the version in the changelog :)

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,12 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
...
+
+### Features
+
+- Add an optional parameter `captchaToken` for captcha support in loginWithPassword
```
Comment (roxanemace) [inline CHANGELOG.md:14]: Don't forget to update the documentation of the method (you can even add a link to the documentation in the changelog).

File: CHANGELOG.md (hunk 2 above)
Comment (FrancoisDevemy5) [reply CHANGELOG.md:10]: Je me disais aussi que c'était trop simple...

**Review by roxanemace — APPROVED** [review]

**Pull request merged**